src/components/referenceChecking.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <p class="comment_user_text" v-if="isReferenceToUser">
+      <span class="comment_user_text_first_word">@{{ firstWord }}</span>
+      {{ commentText.slice(firstSpaceIndex + 1) }}
+    </p>
+    <p class="comment_user_text" v-else>{{ commentText }}</p>
+  </template>
+  
+  <script>
+  import { mapState } from "vuex";
+  
+  export default {
+    name: "ReferenceChecking",
+    props: {
+      commentText: {
+        type: String,
+        required: true,
+      },
+    },
+    computed: {
+      ...mapState({
+        allAccountsList: (state) => state.allAccounts.allAccountsList,
+      }),
+      firstSpaceIndex() {
+        return this.commentText.indexOf(" ");
+      },
+      firstWord() {
+        if (this.commentText[0] === "@") {
+          return this.commentText.substring(1, this.firstSpaceIndex);
+        } else {
+          return "this is not an appeal";
```

Comment:
what does this even mean? so if I call my user `this is not an appeal` then `isReferenceToUser` computed will return `true`... this is a bug